src/util.ts

```diff
@@ -180,17 +172,18 @@ function removeManifestFromSideloadingDirectory(inputApplication: string, manife
       for (let application of Object.keys(applicationProperties)) {
         if (!inputApplication || application === inputApplication) {
           const sideloadingDirectory = applicationProperties[application].sideloadingDirectory;
-          const sideloadingManifestPath = path.join(sideloadingDirectory, path.basename(manifestPathToRemove));
+          const [type, manifestGuid, version] = await parseManifest(manifestPathToRemove);
 
           if (!fs.existsSync(sideloadingDirectory)) {
             continue;
           }
 
           fs.readdirSync(sideloadingDirectory).forEach(manifestName => {
             const realManifestPath = (fs.realpathSync(path.join(sideloadingDirectory, manifestName)));
-            if (sideloadingManifestPath === realManifestPath) {
-              console.log(`Removing ${sideloadingManifestPath} for application ${application}`);
-              fs.unlinkSync(sideloadingManifestPath);
+            const sideloadedManifestPath = path.join(sideloadingDirectory, manifestGuid.concat(".", path.basename(manifestPathToRemove)));
```

Comment:
Same comment as before. Since the same code is in two places, perhaps move it into a single function rather than have duplicate code.

Reply:
created getSideloadManifestPath method